Spinner/Spinner.swift

```diff
@@ -81,6 +86,10 @@ public class SpinnerView: NSObject, Spinner {
             button.userInteractionEnabled = true
             button.restoreTitleColors(controlTitleColors)
         }
+        if let view = spinner?.superview {
+            view.userInteractionEnabled = true
```

Comment:
I am a little concerned about this, since this could alter the behaviour of your app without your consent.
Imagine that you disable user interaction on the superview somewhere in your view controller, then show the spinner without passing `true` as `disablesUserInteraction` parameter. After dismissing the spinner, this would reenable your view user interaction.

One solution for this would be to provide a parameter to the `dismiss()` function, similar to the previous but opposite, so like `enablesUserInteraction` with some default value.

It's the same for the code on line 86 actually with the button, I haven't thought of that previously.